### PR TITLE
added mayhem support

### PR DESCRIPTION
**Pull request by matl4c**

### CODE REVIEW

**Review by xansec — CHANGES_REQUESTED** [review]
Could you please restore the deleted workflow? Please make the requested changes.

**Comment by matl4c** [comment]
changed completed

File: mayhem/fuzz-pywhat.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/python3
```
Comment (xansec) [inline mayhem/fuzz-pywhat.py:1]: One more thing as I'm looking at this - I don't think using `sys.maxsize` is a good idea here, as there's no guarantee on the length of the fuzzed data (could be smaller). Can you instead use `fdp.remainingBytes()` here?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: mayhem/fuzz-pywhat.py (hunk 1 above)
Comment (matl4c) [reply mayhem/fuzz-pywhat.py:1]: change completed

**Review by xansec — APPROVED** [review]
LGTM!

**Comment by xansec** [comment]
@matl4c This appears to not work:

```
 === Uncaught Python exception: ===
AttributeError: 'atheris.native.FuzzedDataProvider' object has no attribute 'remainingBytes'
Traceback (most recent call last):
  File "/src/./mayhem/fuzz-pywhat.py", line 12, in TestOneInput
    data = fdp.remainingBytes()
```

The following commit resolves
https://github.com/mayhemheroes/pyWhat/commit/a78ca10e9ef3f5ff3550662d52f01c68986e30c9

**Comment by matl4c** [comment]
> @matl4c This appears to not work:
> 
> ```
>  === Uncaught Python exception: ===
> AttributeError: 'atheris.native.FuzzedDataProvider' object has no attribute 'remainingBytes'
> Traceback (most recent call last):
>   File "/src/./mayhem/fuzz-pywhat.py", line 12, in TestOneInput
>     data = fdp.remainingBytes()
> ```
> 
> The following commit resolves [a78ca10](https://github.com/mayhemheroes/pyWhat/commit/a78ca10e9ef3f5ff3550662d52f01c68986e30c9)

Sorry, I will test better in the future, thank you for the fix. Is there anything else you need from me on this one?

**Comment by xansec** [comment]
No, that's it for now.

**Pull request merged**